tests/testthat/test-f7Searchbar.R

```diff
@@ -81,8 +81,8 @@ test_that("searchbar works", {
   app$click(selector = "[data-searchbar=\"#search1\"]")
   app$run_js("app.searchbar.search('.searchbar', 'mobile')")
   app$wait_for_idle(2000)
-  app$expect_values(input = "deviceInfo")
```

Comment:
Since `deviceInfo` contains info about OS and more specifics, it leads to different results on GHA. We didn't notice this before because shinytest2 was actually not testing properly 🤷‍♀️